drivers/entropy/entropy_nrf5.c

```diff
@@ -116,10 +116,12 @@ static int random_byte_get(void)
 	return retval;
 }
 
+#if defined(__GNUC__)
 #pragma GCC push_options
 #if defined(CONFIG_BT_CTLR_FAST_ENC)
```

Comment:
`CONFIG_BT_CTLR_FAST_ENC` should likely be documented to be GCC-only (I think we can't create toolchain dependencies in Kconfig)

Reply:
btw, shouldn't we have:

```
#if defined(CONFIG_BT_CTLR_FAST_ENC) && defined(__GNUC__)
#pragma GCC push_options
#pragma GCC optimize ("Ofast")
#endif /* defined(CONFIG_BT_CTLR_FAST_ENC) && defined(__GNUC__) */
```